src/plugins/discover/public/application/main/components/layout/discover_documents.tsx

```diff
@@ -84,8 +84,8 @@ function DiscoverDocumentsComponent({
 
   const onResize = useCallback(
     (colSettings: { columnId: string; width: number }) => {
-      const grid = { ...state.grid } || {};
-      const newColumns = { ...grid.columns } || {};
```

Comment:
I try to remember what the one writing this line thought ... which is me :) ... does not compute ... error ... file not found